src/EnergyPlus/DXCoils.cc

```diff
@@ -10361,14 +10368,14 @@ namespace DXCoils {
         PLRHeating = 0.0;
         DXCoil(DXCoilNum).HeatingCoilRuntimeFraction = 0.0;
         // Initialize crankcase heater, operates below OAT defined in input deck for HP DX heating coil
-        if (OutdoorDryBulb < DXCoil(DXCoilNum).MaxOATCrankcaseHeater) {
+        if (CompAmbTemp < DXCoil(DXCoilNum).MaxOATCrankcaseHeater) {
             CrankcaseHeatingPower = DXCoil(DXCoilNum).CrankcaseHeaterCapacity;
         } else {
             CrankcaseHeatingPower = 0.0;
         }
 
         if ((AirMassFlow > 0.0) && (GetCurrentScheduleValue(DXCoil(DXCoilNum).SchedPtr) > 0.0) && (PartLoadRatio > 0.0) &&
-            OutdoorDryBulb > DXCoil(DXCoilNum).MinOATCompressor) {
+            CompAmbTemp > DXCoil(DXCoilNum).MinOATCompressor) {
```

Comment:
But here I am not exactly sure of which indicator should be used. Should this be looking at the local ambient temp or the water inlet temp? Maybe I got his one wrong (not here, but where I set CompAmbTemp for water cooled), I'm not sure. I think this is only questionable for water cooled.

Reply:
Now you have me second guessing myself on whether this line should be reverted.

Reply:
Turns out this was incorrect. Must compare to condenser inlet temp. Corrected in #7483